drizzlepac/haputils/mvm_quality_analysis.py

```diff
@@ -103,49 +116,94 @@ def report_wcsname(total_product_list, json_timestamp=None, json_time_since_epoc
+                sci_extns = [0]
+
+            # ... and loop over the SCI extensions (each SCI extension is a chip)
+            for sci in sci_extns:
+                wcs = HSTWCS(exp, ext=sci)
+
+                # Get the WCSNAME for this specific exposure/chip
+                wcsname = exp[sci].header['WCSNAME'].upper()
+
+                # Each chip is assigned the 2**n value associated with its WCSNAME
+                suffix = wcsname.split('-')[1]
+                expo_wcs_value = wcs_pref_list[suffix]
+
+                # If this is a multi-chip exposure, get the CHIP number
+                chip_number = 0
+                try:
+                    chip_number = exp[sci].header['CCDCHIP']
+                except:
+                    pass
+
+                # Compute the sky footprint corners for this chip
+                radec = wcs.calc_footprint().tolist()
+                radec.append(radec[0])  # close the polygon/chip
+                ra = [item[0] for item in radec]
+                dec = [item[1] for item in radec]
+
+                # Also save the corner X and Y positions
+                xycorners = metawcs.all_world2pix(radec, 0).astype(np.int32).tolist()
+                xc = [item[0] for item in xycorners]
+                yc = [item[1] for item in xycorners]
+
+                # Load the dictionary with the collected data for this exposure
+                active_wcs_dict = {'filename': edp_object.full_filename,
+                                   'instrument': instrument,
+                                   'detector': detector,
+                                   'primary_wcsname': wcsname,
+                                   'wcs_value': expo_wcs_value,
+                                   'chip_number': chip_number,
+                                   'img_value': expo_img_value,
+                                   'RA': ra,
+                                   'Dec': dec,
+                                   'X': xc,
+                                   'Y': yc}
+
+                # Make the PrimaryWCS unique for every exposure/chip to ensure no entry is overwritten.
+                diagnostic_obj.add_data_item(active_wcs_dict, 'PrimaryWCS' + str(counter) + "_" + str(chip_number), 
+                                             descriptions={'filename': 'Exposure filename',
+                                                           'instrument': 'Instrument',
+                                                           'detector': 'Detector',
+                                                           'primary_wcsname': 'Active WCS',
+                                                           'wcs_value': 'Value (2^n) as numeric ID of WCSNAME',
+                                                           'chip_number': 'CCD Chip Number',
+                                                           'img_value': 'Value (2^n) for fill of the footprint',
```

Comment:
This value could also be used as a 'fill' value for the footprint, but only in a secondary capacity.  Perhaps this comment should be updated to reflect it more as a 'numeric ID' instead?